xml/decoders/MGP_Servo5.xml

```diff
@@ -1074,27 +1957,29 @@
 <!-- * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * *  -->	  
   <pane>
     <name>General decoder info</name>
+    <name xml:lang="cs">Všeobecné informace o dekodéru</name>
     <column>
       <label>
         <text>Address to decoder AND to first servo</text>
+        <text  xml:lang="cs">Adresa dekodéru a prvního serva</text>
       </label>
       <display item="Short Address"/>
       <label>
         <text> </text>
       </label>
-      <display item="Decoder Version" label="Software Version"/>
+      <display item="Decoder Version"/>
       <label>
         <text> </text>
       </label>
-      <display item="Manufacturer Id" label="Manufacturer Id"/>
+      <display item="Manufacturer"/>
       <label>
         <text> </text>
       </label>
-      <display item="Developer Id" label="Developer Id"/>
+      <display item="Developer Id"/>
       <label>
         <text> </text>
       </label>
-      <display item="Product Id" label="Product Id"/>
+      <display item="Product Id"/>
```

Comment:
@bobjacobsen @dheap 
Can any of you confirm that this is the correct change? The rest of the PR looks good to me, but I'm uncertain about these changes.

Reply:
Dropping a label that's identical to the `item` value is a NOP, so the 1st, 3rd and 4th are OK for that reason.

The proper item value is "Manufacturer" (see the names.xml file), so the 2nd change is actually a bug fix.

Hence all OK.

Reply:
Thanks!